src/components/TimeLine/Timeline.jsx

```diff
@@ -0,0 +1,67 @@
+import { VerticalTimeline, VerticalTimelineElement }  from 'react-vertical-timeline-component';
+import 'react-vertical-timeline-component/style.min.css';
+import './TimeLine.modules.css'
+import '../../img/search.svg'
+import SearchIcon from '@mui/icons-material/Search';
```

Comment:
acho que entendi, você tá usando MUI pra pegar os icones e a MUI deve precisar do emotion. Vou baixar e dar uma olhada mas se for isso acho que deveriamos mudar os icones do bootstrap, mas vou dar uma olhada aqui se eh possivel e se realmente tem impacto no tamanho do nosso app.